### PR TITLE
Optimize the components and integrated modal in editing profile

### PR DESCRIPTION
**Pull request by janrebpol**
**OVERVEIEW**
Organized the files for each component of the UI in the newsfeed and integrated the modal for Edit Profile.

**SNIPPETS**
image
image
image

### CODE REVIEW

File: src/resources/css/app.css [hunk 1]
```diff
@@ -3,3 +3,19 @@
...
+        display: none;
+      }
+
+
+      /* Hide scrollbar for IE, Edge, and Firefox */
+      .no-scrollbar {
+        -ms-overflow-style: none; /* IE and Edge */
+        scrollbar-width: none; /* Firefox */
+      }
+    }
```
Comment (hipe01mark) [inline src/resources/css/app.css]: Are those comments necessary?

File: src/resources/views/components/feeds/profile-icon.blade.php [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        <div class="text-mydark text-sm font-semibold">Moon</div>
+        <div class="flex w-full mt-1 ">
+            <div class="text-mydark flex font-light text-xs gap-2">
+                • 30 seconds ago <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" fill="currentColor"
+                    class="w-4 h-4 text-mygray cursor-pointer hover:fill-mydark">
+                    <path
+                        d="m5.433 13.917 1.262-3.155A4 4 0 0 1 7.58 9.42l6.92-6.918a2.121 2.121 0 0 1 3 3l-6.92 6.918c-.383.383-.84.685-1.343.886l-3.154 1.262a.5.5 0 0 1-.65-.65Z" />
+                    <path
+                        d="M3.5 5.75c0-.69.56-1.25 1.25-1.25H10A.75.75 0 0 0 10 3H4.75A2.75 2.75 0 0 0 2 5.75v9.5A2.75 2.75 0 0 0 4.75 18h9.5A2.75 2.75 0 0 0 17 15.25V10a.75.75 0 0 0-1.5 0v5.25c0 .69-.56 1.25-1.25 1.25h-9.5c-.69 0-1.25-.56-1.25-1.25v-9.5Z" />
+                </svg>
```
Comment (hipe01mark) [inline src/resources/views/components/feeds/profile-icon.blade.php]: Create a seperate file for sgv

File: src/resources/css/app.css (hunk 1 above)
Comment (janrebpol) [reply src/resources/css/app.css]: I forgot to delete them; sir, they are no longer needed.

File: src/resources/views/components/feeds/profile-icon.blade.php (hunk 2 above)
Comment (janrebpol) [reply src/resources/views/components/feeds/profile-icon.blade.php]: Separated the SVGs , also added follow suggestions.

**Review by hipe01mark — APPROVED** [review]
LGTM 🙇

**Pull request merged**